go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/kserve/rest-proxy
 
-go 1.19
+go 1.21
```

Comment:
Did you run `go mod tidy -compat=1.21` to update the dependencies below?

Reply:
just go mod tidy

Reply:
Just did, nothing else changed.

```
❯ go mod tidy -compat=1.21
❯ git status
On branch go1.21                                                                                                                                               ─╯
nothing to commit, working tree clean
```

Reply:
cool. thanks